fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/jbatch/export/patient/PatientExportPartitionMapper.java

```diff
@@ -36,6 +36,8 @@
     @Inject
     JobContext jobCtx;
 
+    private static final CompartmentUtil compartmentHelper = new CompartmentUtil();
```

Comment:
Not a proper singleton pattern, but I assume it doesn't matter if multiple copies exist.

Reply:
I debated whether to use a singleton pattern for CompartmentUtil or not.  The initialization logic does depend on the state of the registry and so if we did make it a singleton then I think we'd want a way to "re-initialize" it with the latest from the registry

Reply:
Yeah, just want to make sure that we're not concerned about this being a strict singleton.